package.json

```diff
@@ -34,10 +33,9 @@
         "typescript": "~3.9.7"
     },
     "dependencies": {
-        "@aws-cdk/aws-eks": "1.135.0",
-        "@aws-cdk/aws-iam": "1.135.0",
-        "@aws-cdk/core": "1.135.0",
-        "@aws-quickstart/ssp-amazon-eks": "^0.12.3",
+        "@aws-quickstart/eks-blueprints": "^0.1.1",
+        "aws-cdk": "2.17.0",
```

Comment:
This package is called `aws-cdk-lib` not `aws-cdk`
`aws-cdk` is a CLI toolkit I believe.